src/offset/local/win_bindings.txt

```diff
@@ -0,0 +1,6 @@
+--out src/offset/local/win_bindings.rs
+--config flatten sys
```

Comment:
Have you considered the non-sys bindings that `windows-bindgen` can now generate, with a cleaner API and helpers? The overhead in `windows-core` and the extra functions/types that may need to be generated might not be worth it though, but it's good to have that documented.

Reply:
Yes, that plan makes sense to me.